examples/pytorch-mnist-membership-inference/docker-compose.yml

```diff
@@ -202,12 +207,13 @@ services:
       KMP_SETTINGS: "FALSE"
       OMP_PROC_BIND: "false"
       RQ_REDIS_URI: redis://redis:6379/0
+      TF_CPP_MIN_LOG_LEVEL: "2"
     command:
       - pytorch_cpu
     networks:
       - lab_deployment_backend
     volumes:
-      - nfs-datasets:/nfs/data/Mnist
+      - nfs-datasets:/nfs/data
```

Comment:
If the `demo.ipynb` notebook is reverted back (see my comment on `demo.ipynb` file), then this file should reverted to https://github.com/usnistgov/dioptra/blob/29ee50438a001715d1ca0f3038527fe286e74719/examples/pytorch-mnist-membership-inference/docker-compose.yml to ensure compatibility.